src/xenia/kernel/xam/user_profile.cc

```diff
@@ -107,7 +115,56 @@ UserProfile::Setting* UserProfile::GetSetting(uint32_t setting_id) {
   if (it == settings_.end()) {
     return nullptr;
   }
-  return it->second;
+  UserProfile::Setting* setting = it->second;
+  if (setting->is_title_specific()) {
+    if (kernel_state()->title_id() != setting->loaded_title_id) {
```

Comment:
I'm confused - why is this a `!=` comparison?

Reply:
The idea is that we only load from disk once, and after that it is already in memory.  Once it is in memory, we only need to load from disk if we're now running a different title.  Thus the check here is to see if what we've got in memory is for the title that is currently running.  I'll add a comment explaining.

Reply:
We could just always load it from disk if we're not worried about games spamming the load though.  None of the games I've tried did it more than once.